src/wp-includes/class-wp-block-type-registry.php

```diff
@@ -96,6 +96,15 @@ public function register( $name, $args = array() ) {
 
 		$this->registered_block_types[ $name ] = $block_type;
 
+		/**
+		 * Fires after a block type is registered.
+		 *
+		 * @since 6.1.0
+		 *
+		 * @param WP_Block_Type|false $block_type The registered block type on success, or false on failure.
+		 */
+		do_action( 'registered_block_type', $block_type );
```

Comment:
I'm not completely sure of what the best naming convention is, but I see some actions use past tense and others use something like 'after_register_block_type' or 'post_register_block_type'.

It would be interesting to know if there's a preference!

Reply:
> It would be interesting to know if there's a preference!

Same, I'd be curious to hear what folks think! In this case, I borrowed the naming convention from `registered_post_type` and `registered_taxonomy`.